src/components/Widget/index.tsx

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line
```

Comment:
This should include a comment preceding the eslint directive explaining why it is necessary.
Less importantly, it should also explicitly disable only the necessary lint rule.

```
// Import fonts.css for the side-effect of loading fonts for @uniswap/widgets.
// eslint-disable-next-line i-don't-know-the-rule
```